### DISCUSSION TITLE
View Coin List from GUI

### DISCUSSION THREAD

**Discussion by nezzee** [Chia-Network/chia-blockchain-gui#2376] [Ideas]
UTXO appears to be a concept that many more casual users do not grasp with what is happening in the background (especially because the presentation is that of an account model with the balance).

I think that it might help users better understand what is happening to have a breakdown page in the wallet (per token) to better understand the coin composition of their wallet in a more visual way.  Perhaps even breaking down into groups based on wallet addresses, or percentages of the whole, or various other implementations.

I don't believe it needs to be front and center, but at least an option to click into that helps as a learning tool, as well as a way to maybe easily combine or split coins, or view all the different addresses you possess if you want to rebalance to new addresses.

Just a bit more of an easy way for a user to have more control into how their wallet is comprised, that way, users can be a bit more understanding as to what is physically happening on a tech level (such as the regular "why is my receive address different" or "why can't I send my full balance" or "what is dust").

**Comment by BrandtH22** [comment]
@nezzee , note I transferred this enhancement request to the GUI repo for better visibility 👍
